storage/volume/core_test.go

```diff
@@ -0,0 +1,53 @@
+package volume
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/alibaba/pouch/storage/volume/driver"
+	"github.com/alibaba/pouch/storage/volume/types"
+)
+
+func createVolumeCore(root string) (*Core, error) {
+	cfg := Config{
+		VolumeMetaPath: path.Join(root, "volume.db"),
+	}
+
+	return NewCore(cfg)
+}
+
+func TestCreateVolume(t *testing.T) {
+	dir, err := ioutil.TempDir("", "test-create")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(dir)
+
+	// create volume core
+	core, err := createVolumeCore(dir)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	driver.Register(driver.NewFakeDriver("fake1"))
```

Comment:
Use a variable like `fakeDriverName` instead of hard coding "fake1". @shaloulcy

Reply:
we have two drivers: **fake1** and **none**. use fakeDriverName and noneDriverName?

Reply:
agree

Reply:
Maybe I will add a driver named **fake2** or **fake3** in List interface, then? sometimes I prefer hard coding.

Reply:
You can take https://github.com/alibaba/pouch/blob/master/pkg/utils/utils_test.go#L127 for reference. Table driven test. @shaloulcy